### PR TITLE
renamed "Secondary Stats" to "Capabilities" to clarify wording

### PR DESCRIPTION
**Pull request by bleehu**
modified:   01_01_Basic_Rules.txt
	modified:   03_Glossary_of_Terms.txt
	modified:   CharacterCreation/00_00_Creating_a_character.txt
	modified:   Items/05_Armors.txt

### CODE REVIEW

**Comment by bleehu** [comment]
I'd also like to take the opportunity to point out that having move speed be a per-2-action system makes test users very confused. I'd like to fix that later.

**Review by Reveraine — APPROVED** [review]
I like this change! Ship it!

File: 01_01_Basic_Rules.txt [hunk 1]
```diff
@@ -379,10 +386,10 @@ object depending on how heavy it is.
 
 ===== Nanites =====
 
-	Although not a primary or secondary stat per se, The maximum number of 
-Nanites you can use at a given time is determined by your character class and an 
-associated primary stat. Your class may tell you its "Class Primary Attribute" 
-is Strength. In that case, Strength * 10 is your maximum Nanites. 
+	The maximum number of Nanites you can use at a given time is determined by 
+your character class and an associated Primary Stat. Your class may tell you its 
+"Class Primary Attribute" is Strength. In that case, Strength * 10 is your
```
Comment (Turtlelord26) [inline 01_01_Basic_Rules.txt]: This is obsolete, please scope creep to kill it?

Comment (Turtlelord26) [reply 01_01_Basic_Rules.txt]: I thought I changed all the references to old nanites?

**Review by Turtlelord26 — APPROVED** [review]
Looks good, subject to later terminology review. 

A clarity pass on move speed is very much green lit.

File: 01_01_Basic_Rules.txt (hunk 1 above)
Comment (bleehu) [reply 01_01_Basic_Rules.txt]: I was wondering about that. Ya, I'll grab it.

**Comment by bleehu** [comment]
Fixed Issue #714
